pkg/reconciler/route/domains/domains.go

```diff
@@ -67,3 +87,26 @@ func URL(scheme, fqdn string) *apis.URL {
 		Path:   fqdn,
 	}
 }
+
+// RouteDomain will generate the Route's Domain(host) for the Service based on
+// the "DomainTemplateKey" from the "config-network" configMap.
+func RouteDomain(ctx context.Context, route *v1alpha1.Route) (string, error) {
```

Comment:
This function looks identical to `DomainNameFromTemplate`

Reply:
Done